app/assets/javascripts/controllers/ExamFormCtrl.coffee

```diff
@@ -27,6 +28,18 @@ angular.module('infish').controller 'ExamFormCtrl', ($scope, $state, Exam) ->
   $scope.addNewQuestion = ->
     pushNewQuestion()
 
+  $scope.addNewFilledQuestion = (question, answers, states) ->
```

Comment:
`question` variable is confusing, since we are adding new question at this function. What do you think about `question_text`?